_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: drupal drupal8 drupal-planet marketing 
+summary: SEO trends 2017 and Drupal 8 as the best option for your long-term marketing campaign.
+featured_image: "/assets/img/blog/supercharged-seo-with-drupal-8.jpg"
+featured_image_alt: "Supercharged SEO with Drupal 8"
+---
+
+Here at Savas Labs, we listen to our clients needs, and what many of our clients need is to reach their target audiences effectively. Let’s be honest here - perfectly coded, a pretty looking website will be of little use if it doesn’t produce leads / facilitate conversions or generate revenue! So how do we help our clients achieve their goals? We get there by balancing website objectives while giving priority to lead generation via SEO. The more quality traffic that comes to the website - the more conversions we can achieve. It’s that simple!
+
+There are multiple ways of generating traffic to a website. The most popular methods are SEO (Search Engine Optimization) and PPC (Pay Per Click). Both bring traffic through search engines. The difference between the two is that SEO brings long-term results boosting organic traffic while PPC helps marketers achieve short-term goals by gaining instant exposure throughout the duration of an ad campaign.
+
+In this post I’ll share some insight about current SEO trends. I’ll also describe new features of  Drupal 8 that make it the most SEO-friendly content management framework available today.
+
+Let’s start by taking a look at what exactly Search Engine Optimization is.
+
+## What is SEO?
+
+Search Engine Optimization (SEO) is a marketing discipline focused on optimizing a website’s architecture and content so that it performs well (read “ranks high”) in organic search engine results.
```

Comment:
It looks like there are "smart quotes", e.g. `“` instead of `"`, and `’`, instead of `'` -- I would recommend converting these to (for lack of a better term) "dumb quotes" :smile: